java-extensions/hive-reader/src/main/java/com/starrocks/hive/reader/HiveColumnValue.java

```diff
@@ -181,4 +165,16 @@ public byte getByte() {
     public BigDecimal getDecimal() {
         return ((HiveDecimal) inspectObject()).bigDecimalValue();
     }
+
+    @Override
+    public LocalDate getDate() {
+        return LocalDate.ofEpochDay((((DateObjectInspector) fieldInspector).getPrimitiveJavaObject(fieldData))
+                .toEpochDay());
+    }
+
+    @Override
+    public LocalDateTime getDateTime(ColumnType.TypeValue type) {
+        return LocalDateTime.ofInstant(Instant.ofEpochSecond((((TimestampObjectInspector) fieldInspector)
+                .getPrimitiveJavaObject(fieldData)).toEpochSecond()), ZoneId.of(timeZone));
```

Comment:
is this compatible with hive 2?

https://github.com/StarRocks/starrocks/pull/37185

Reply:
done